src/RoundTabler/PCIScan.java

```diff
@@ -13,12 +16,66 @@
 	// with embedded dashes
 	static String CardNumberSequenceRegex = "\b(?:\\d[-]*?){13,16}\b";
 	static Pattern CardNumberPattern = Pattern.compile( CardNumberSequenceRegex);
+	
+	public String JDBC_DRIVER;
+
+	public string ScanResult;
```

Comment:
this needs to be updated to "String" not "string"

Reply:
Fixed